cores/esp32/esp32-hal-spi.c

```diff
@@ -1059,7 +1059,7 @@ void spiSimpleTransaction(spi_t * spi)
     if(!spi) {
         return;
     }
-    SPI_MUTEX_LOCK();
+	SPI_MUTEX_UNLOCK();
```

Comment:
Possible bug _LOCK() changed to _UNLOCK().  I will do a more in depth study of the code this weekend.

Reply:
it should be LOCK as that is another way to start transaction